core/Scene.js

```diff
@@ -45,19 +45,55 @@ var Size = require('./Size');
  *                 the renderers and update nodes in the scene graph
  */
 function Scene (selector, updater) {
+    var _this = this;
+
     if (!selector) throw new Error('Scene needs to be created with a DOM selector');
     if (!updater) throw new Error('Scene needs to be created with a class like Famous');
 
     Node.call(this);         // Scene inherits from node
 
-    this._updater = updater; // The updater that will both
-                             // send messages to the renderers
-                             // and update dirty nodes 
+    this._initialized = false;
+
+    // The updater that will both send messages to the renderers and update
+    // dirty nodes
+    this._updater = Object.create(updater);
```

Comment:
We really shouldn't be creating a new FamousEngine here, only to overwrite its methods a few lines down.